README.md

```diff
@@ -55,11 +57,21 @@ Operations are the raw instructions the store uses to make modifications to the
 Each operation is a simple object which contains instructions with the `OperationType`, `path` and optionally the `value` (depending on operation).
 
 ```ts
-const operations = [
-	{ op: OperationType.ADD, path: new JsonPointer('/foo'), value: 'foo' },
-	{ op: OperationType.REPLACE, path: new JsonPointer('/bar'), value: 'bar' },
-	{ op: OperationType.REMOVE, path: new JsonPointer('/qux') },
-];
+import { OperationType } from '@dojo/stores/State/patch';
+import { Pointer } from '@dojo/stores/state/Pointer';
```

Comment:
If you're going to import `Pointer` probably want to change the `new JsonPointer` lines :)

Reply:
Ha oops, fixed